scripts/zkai_native_d128_two_slice_outer_statement_gate.py

```
@@ -0,0 +1,432 @@
+    if not result["all_mutations_rejected"]:
+        raise OuterStatementGateError("one or more mutations were accepted")
+    return result
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--write-json", type=pathlib.Path, default=JSON_OUT)
+    parser.add_argument("--write-tsv", type=pathlib.Path, default=TSV_OUT)
+    args = parser.parse_args()
+    if args.write_json == args.write_tsv:
+        raise SystemExit("write-json and write-tsv output paths must be distinct")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)